.github/workflows/ci.yml

```
@@ -82,6 +86,15 @@ jobs:
           ./cpm findings --learn | head -20
           ./cpm findings --compliance OWASP | head -10
         shell: bash
+      - name: Smoke test (Windows)
+        if: (needs.changes.outputs.code == 'true' || github.ref == 'refs/heads/main') && matrix.os == 'windows-latest'
+        run: |
+          ./cpm.exe help
+          ./cpm.exe version
+          ./cpm.exe score
+        shell: bash
```

actionlint (1.7.12)

[error] shell name "bash ./cpm findings --compliance OWASP | head -10" is invalid. available names are "bash", "cmd", "powershell", "pwsh", "python", "sh"

(shell-name)